src/core/EntryAttributes.cpp

```diff
@@ -30,6 +30,9 @@ const QString EntryAttributes::URLKey = "URL";
 const QString EntryAttributes::NotesKey = "Notes";
 const QStringList EntryAttributes::DefaultAttributes(QStringList()
                                                      << TitleKey << UserNameKey << PasswordKey << URLKey << NotesKey);
+const QString EntryAttributes::UuidKey = "Uuid";
+const QString EntryAttributes::TagsKey = "Tags";
+const QStringList EntryAttributes::TopLevelAttributes(QStringList() << UuidKey << TagsKey);
```

Comment:
I am not a fan of defining these here since they are, as you said, not attributes. These are only used in the cli so I think it is OK to just hard code that behavior (dump data) in the cli code.

Reply:
@droidmonkey I agree. I moved the field name handling to `cli/Utils`.